ecrtokenrefresher/pkg/kubernetes/kubernetes.go

```diff
@@ -0,0 +1,142 @@
+package k8s
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"os"
+	"path/filepath"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+type config struct {
+	Auths map[string]*auth `json:"auths"`
+}
+
+type auth struct {
+	Username string `json:"username"`
+	Password string `json:"password"`
+	Email    string `json:"email"`
+	Auth     string `json:"auth"`
+}
+
+const (
+	defaultEmail      = "test@test.com"
+	configMapName     = "ns-secret-map"
+	packagesNamespace = "eksa-packages"
+)
+
+func UpdatePasswords(name string, username string, token string, registries string) (error, []string) {
```

Comment:
```suggestion
func UpdateTokens(name string, username string, token string, registries string) (error, []string) {
```

The order of the args seems a bit odd to me